stock_inventory_valuation_report/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Stock Inventory Valuation Report',
+    'summary': 'Add Inventory Valuation Report on Inventory Reporting.',
+    'version': '12.0.1.0.0',
+    'category': 'Reporting',
```

Comment:
Please use a standard category. It will help display on OCA shop.

![image](https://user-images.githubusercontent.com/19529533/63407007-c87ee100-c3eb-11e9-8780-59e96bb7bace.png)

Reply:
Thank you for suggestion.